api/metric/common.go

```diff
@@ -48,12 +48,12 @@ func (m commonMetric) Impl() InstrumentImpl {
 	return m.instrument
 }
 
-func (h commonHandle) recordOne(ctx context.Context, number core.Number) {
-	h.handle.RecordOne(ctx, number)
+func (h commonBoundInstrument) recordOne(ctx context.Context, number core.Number) {
```

Comment:
One thing I noticed is that `recordOne` is private in `commonBoundInstrument`, meaning that `commonBoundInstrument` does not implement `BoundInstrumentImpl`.

Is that a bug or am I missing something here?

Reply:
This method has a confusing name -- but it's used to implement the Add(), Set(), and Record() methods of the un-bound instruments.  I would support a rename to something like `directRecord`.